src/nomad_simulations/model_system.py

```diff
@@ -283,12 +318,216 @@ def normalize(self, archive, logger) -> None:
         super().normalize(archive, logger)
 
 
+def convert_combo_to_name(combo: tuple[str]) -> str:
+    """Convert a combo tuple to a hypen-separated string.
+    For angles, the center element is now indeed placed at the center."""
+    if 1 < len(combo) < 4:
+        if len(combo) == 3:
+            combo = (combo[1], combo[0], combo[2])  # flip order
+        return '-'.join(combo)
+    raise ValueError('Only pairs and triplets are supported.')
```

Comment:
The angle distribution format is set up in `DistributionFactory`, where it is indeed defined as such (see `get_elemental_triples_centered`). I could see to define a `dict` mapping or class to handle the definition.

It's not meant to be passed from a parser, at least in the first instance. Do you see a scenario where this could apply?

Reply:
No, that was really just what I wanted to know. Is the order of the atoms with respect to their position in the angle handled somewhere else so that you can be sure the central atom is always listed at the first position here? That does seem to be the case, so it's all good.

Reply:
Okay! I understand that the semantics of the format could be emphasized with a mapping or class, but it looks kinda like overkill to me. Let's just keep it in mind.

I did further specialize `GeometryDistribution`, where the semantics are clearly specified.